editor/src/Editor.cpp

```diff
@@ -76,7 +81,18 @@ namespace Wraith
 
         m_CameraEntity.AddComponent<NativeScriptComponent>().Bind<CameraController>(10.f);
 
-        m_SceneHierarchyPanel.SetContext(m_ActiveScene);
+        {
+            auto play_panels = m_PanelManager.CreateGroup("Play");
+
+            play_panels->CreatePanel<BarPanel>([this] { OnScenePlay(); }, [this] { OnSceneEndPlay(); });
+            m_ViewportPanel = play_panels->CreatePanel<ViewportPanel>(*m_Engine->GetRenderer(), m_CameraController);
+
+            auto scene_panels = m_PanelManager.CreateGroup("Scene");
```

Comment:
Might be an idea to save these panels, since we might wanna add stuff to them 🤔 But it's fine as is right now

Reply:
groups*